src/main/java/org/folio/service/budget/BudgetService.java

```diff
@@ -123,16 +130,18 @@ private Future<Void> unlinkGroupFundFiscalYears(String id, DBClient client) {
     client.getPgClient()
       .execute(client.getConnection(), sql, Tuple.of(UUID.fromString(id)), reply -> {
         if (reply.failed()) {
-          logger.error("Failed to update group_fund_fiscal_year by budgetId={}", id, reply.cause());
+          logger.error("unlinkGroupFundFiscalYears:: Failed to update group_fund_fiscal_year by budget id {}", id, reply.cause());
           handleFailure(promise, reply);
         } else {
+          logger.info("unlinkGroupFundFiscalYears:: Group fund fiscal years have been successfully unlinked");
```

Comment:
such info logs useful only if we have some identifier in them

Reply:
added id